docs/application/native/guides/text-input/autofill-manager.md

```diff
@@ -0,0 +1,174 @@
+# Autofill Manager
+
+Tizen autofill framework allows more than one or more autofill services to be installed in a Tizen device. Autofill manager is a module for managing the installed autofill services. Application developers can use autofill manager to get the current autofill service or change the autofill service.
+
+The main features of the Autofill Manager API include:
+
+- [Getting the installed autofill service list](#getting-the-installed-autofill-service-list)
+
+  Use the `autofill_manager_foreach_autofill_service()` function to get the installed autofill service list in your device.
+  As a result, the `autofill_manager_autofill_service_cb()` callback function is invoked and you can get the installed autofill services.
+
+- [Getting the current autofill service](#getting-the-current-autofill-service)
+
+  Use the `autofill_manager_get_autofill_service()` function to get the current autofill service.
+
+- [Setting the autofill service](#setting-the-autofill-service)
+
+  Use the `autofill_manager_set_autofill_service()` function to set the autofill service.
+
+## Prerequisites
+
+1. To use the Autofill Manager API (in [mobile](../../api/mobile/latest/group__CAPI__UIX__AUTOFILL__MANAGER__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__UIX__AUTOFILL__MANAGER__MODULE.html) applications), the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+
+   ```xml
+   <privileges>
+      <privilege>http://tizen.org/privilege/autofillmanager</privilege>
+   </privileges>
+   ```
+
+2. To use the functions and data types of the Autofill Manager API, include the `<autofill_manager.h>` header file in your application:
+
+    ```c
+    #include <autofill_manager.h>
+    ```
+
+3. To use the Autofill manager library, create an autofill manager handle.
+
+   The autofill manager handle is used in other autofill manager functions as a parameter.
+
+   ```c
+   void
+   create_autofill_manager_handle()
+   {
+       autofill_manager_h amh;
+       int ret;
+       ret = autofill_manager_create(&amh);
+       if (ret != AUTOFILL_ERROR_NONE)
+           /* Error handling */
+   }
+   ```
+
+4. When you no longer need the autofill manager library, destroy the autofill manager handle using the `autofill_manager_destroy()` function:
```

Comment:
Please verify the modified sentence:
Incase, the Autofill manager library is not needed in your application, you must destroy the Autofill manager handle using the
`autofill_manager_destroy()` function: